src/lib/adamant-api/index.d.ts

```diff
@@ -90,6 +90,10 @@ export type SendMessageParams = {
 }
 export function sendMessage(params: SendMessageParams): Promise<CreateNewChatMessageResponseDto>
 
+export function signChatMessageTransaction(params: SendMessageParams): Promise<RegisterAnyTransaction>
```

Comment:
```suggestion
export function signChatMessageTransaction(params: SendMessageParams): Promise<RegisterAnyTransaction>
```

I think the response type `RegisterAnyTransaction` is not correct here

Reply:
RegisterChatMessageTransaction, see schema.adamant.im